test/integration/edition_edit_test.rb

```diff
@@ -253,6 +267,56 @@ class EditionEditTest < IntegrationTest
         end
       end
 
+      context "when state is 'published'" do
+        context "content type is retired" do
+          setup do
+            visit_retired_edition_in_published
+            click_link("Admin")
+          end
+
+          should "not show the 'Update content type' form" do
+            assert page.has_no_text?("Update content type")
+          end
+        end
+
+        context "edition is not the latest version of a publication" do
+          setup do
+            visit_old_edition_of_published_edition
+            click_link("Admin")
+          end
+
+          should "not show the 'Update content type' form" do
+            assert page.has_no_text?("Update content type")
+          end
+        end
+
+        context "content type is not retired, edition is the latest version of a publication" do
+          setup do
+            visit_published_edition
+            click_link("Admin")
+          end
+
+          should "show the 'Update content type' form" do
+            assert page.has_text?("Update content type")
+          end
+
+          should "show radio buttons for all content types excluding current one (answer)" do
+            assert page.has_no_selector?(".gem-c-radio input[value='answer']")
+            assert page.has_selector?(".gem-c-radio input[value='completed_transaction']")
+            assert page.has_selector?(".gem-c-radio input[value='guide']")
+            assert page.has_selector?(".gem-c-radio input[value='help_page']")
+            assert page.has_selector?(".gem-c-radio input[value='place']")
+            assert page.has_selector?(".gem-c-radio input[value='simple_smart_answer']")
+            assert page.has_selector?(".gem-c-radio input[value='transaction']")
+          end
+
+          should "show explanatory text content for all content types except Guides" do
```

Comment:
Maybe we can reword this as to me the name here implies that we're checking that the text is displayed for all content types other than guides, rather than we're checking that the text common to all content types is displayed and the guide-specific text is not displayed.